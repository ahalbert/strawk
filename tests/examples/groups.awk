# hello
/([^,]+),([^,\n]+)*/ {
  for (g in $MATCHES) {
    print g, $MATCHES[g]
  }
}

