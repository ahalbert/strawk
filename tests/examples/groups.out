$0 hello,world
$1 hello
$2 world
